app/controllers/content_types_controller.rb

```diff
@@ -38,13 +38,51 @@ def render_error(msg)
 
   def cocina_update_attributes
     {}.tap do |attributes|
-      attributes[:type] = Constants::CONTENT_TYPES[params[:new_content_type]] if content_type_should_change?
-      attributes[:structural] = structural_with_resource_type_changes if resource_types_should_change?
+      attributes[:type] = Constants::CONTENT_TYPES[new_content_type] if content_type_should_change?
+      attributes[:structural] = if resource_types_should_change?
+                                  structural_with_resource_type_changes
+                                else
+                                  @cocina_object.structural.new(hasMemberOrders: member_orders)
+                                end
     end
   end
 
+  def old_content_type
+    if @cocina_object.type == 'http://cocina.sul.stanford.edu/models/book.jsonld' && @cocina_object.structural.hasMemberOrders[0]&.viewingDirection == 'right-to-left'
+      # if we have a book type, we need to also check the viewing direction
+      'book (rtl)'
```

Comment:
Can you add a test for this case and for (ltr)?  It's not immediately clear in `constants.rb` that the order of the members is important, but it is to this method.

Reply:
Yes, good point.  I added several tests to the `content_types_controller_spec` to verify the `old_content_type` method works as expected in various scenarios